include/inviwo/qt/editor/inviwomainwindow.h

```diff
@@ -113,18 +115,23 @@ class IVW_QTEDITOR_API InviwoMainWindow : public QMainWindow, public NetworkEdit
     bool askToSaveWorkspaceChanges();
     void exitInviwo(bool saveIfModified = true);
     void showAboutBox();
+    
+    void showWelcomeScreen();
+    void hideWelcomeScreen();
 
 protected:
     virtual void dragEnterEvent(QDragEnterEvent* event) override;
     virtual void dragMoveEvent(QDragMoveEvent* event) override;
     virtual void dropEvent(QDropEvent* event) override;
 
 private:
+    friend WelcomeWidget;
+
     virtual void onModifiedStatusChanged(const bool& newStatus) override;
 
     void visibilityModeChangedInSettings();
 
-    void openWorkspace(QString workspaceFileName, bool exampleWorkspace);
+    bool openWorkspace(QString workspaceFileName, bool exampleWorkspace);
```

Comment:
Returns true if?
We might change "exampleWorkspace" to "readOnly" since that it what we mean?